client/src/components/SentimentAnalysis/SimilarAds.js

```diff
@@ -0,0 +1,55 @@
+import { database } from '../../firebase/firebase';
+import React, { useState, useEffect } from 'react';
+
+const SimilarAds = props => {
+
+  const COLLECTION = 'ads';  
+  const headlineSentiment = `{"score": ${props.headlineScore}, "magnitude": ${props.headlineMagnitude}}`;
+  const contentSentiment = `{"score": ${props.contentScore}, "magnitude": ${props.contentMagnitude}}`;
+
+  const [results, setResults] = useState([]);
+
+  useEffect(() => {
+    async function fetchSimilarAds() {
+      let similarAds = await database
+          .collection(COLLECTION)
+          .where('headlineSentiment', '==', headlineSentiment)
+          .where('contentSentiment', '==', contentSentiment)
+          .limit(9)
+          .get();
+      const resultItems = [];
+      for (let ad of similarAds.docs) {
+        resultItems.push(
+          <div className="similar-ad">
+            <p><b>Headline</b>: {ad.data().headline}</p>
```

Comment:
This is the nitpickiest of nitpicks, but just as an educational fyi, when you use tags like \<b\> and \<strong\> SEO bots assign semantic meaning to them and think they're the most important parts of your website.  Since that isn't the case here, using a \<span\> with font-weight: 700 would technically be more correct.

This is true in general of tags -- HTML tags represent semantic meaning.  There is a pretty big difference between \<h1\> and \< span style="font-size: 32px, font-weight: 700" \> in terms of how bots and rendering/interpreter tools will interpret your website.